### PR TITLE
update

### PR DESCRIPTION
**Pull request by PriyankaChaganti**

### CODE REVIEW

File: robust_svm/main.py [hunk 1]
```diff
@@ -23,10 +23,10 @@
 
 # Process the training images and load the training data
 print('Loading training data')
-training_data = make_dataset(settings.training_data_folder, feature_path, data_set_list)
-pickle.dump(training_data, open(training_data_pickle_path, "wb"))
+#training_data = make_dataset(settings.training_data_folder, feature_path, data_set_list)
```
Comment (sahithi403) [inline robust_svm/main.py]: don't comment these lines in the PR

File: robust_svm/main.py [hunk 2]
```diff
@@ -23,10 +23,10 @@
...
+#pickle.dump(training_data, open(training_data_pickle_path, "wb"))
 # Comment the above two lines and uncomment the following line to avoid re-processing images
-#training_data = pickle.load(open(training_data_pickle_path, "rb"))
+training_data = pickle.load(open(training_data_pickle_path, "rb"))
```
Comment (sahithi403) [inline robust_svm/main.py]: do not uncomment this in the PR

File: robust_svm/main.py [hunk 3]
```diff
@@ -61,6 +61,10 @@
 ###########################################################################################
 accuracy_results = test_classifier(multi_svm_classifier,test_data)
 print("Accuracy Results:")
+sum = 0
```
Comment (sahithi403) [inline robust_svm/main.py]: sum is a terrible name for a variable. It shadows the builtin function 'sum'. Did you not get a warning from pycharm?

**Comment by sahithi403** [comment]
@PriyankaChaganti Did you run the new files? Did everything run fine?

File: robust_svm/main.py (hunk 3 above)
Comment (PriyankaChaganti) [reply robust_svm/main.py]: no

**Review by sahithi403 — APPROVED** [review]

**Pull request merged**